components/feature/app-links/src/main/java/mozilla/components/feature/app/links/AppLinksUseCases.kt

```diff
@@ -218,4 +232,16 @@ class AppLinksUseCases(
         val marketplaceIntent: Intent? = null,
         val resolveInfo: ResolveInfo? = null
     )
+
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    internal data class AppLinkRedirectCache(
+        var cacheTimeStamp: Long = 0,
```

Comment:
nit: Remove the default arguments. I don't think it makes sense to have defaults for a cache now that we've made the property nullable.
```suggestion
        var cacheTimeStamp: Long,
        var cachedUrlHash: Int,
        var cachedAppLinkRedirect: AppLinkRedirect
```

Reply:
Agreed.  I'll remove.  Thanks,